### PR TITLE
Adding docs for excluding a step

### PR DESCRIPTION
**Pull request by rheinwein**
The base functionality is in Jet, but the UI updates have not been pushed to Mothership.

https://www.pivotaltracker.com/story/show/106991536

### CODE REVIEW

File: _posts/docker/2015-05-25-steps.md [hunk 1]
```diff
@@ -42,23 +42,27 @@ Steps are specified in a list, such as:
   command: echo bar
 ```
 
-Steps are executed in the order they are specified. All steps share two directives:
+Steps are executed in the order they are specified. All steps share these directives:
 
 * `name` specifies the name of the step, outputted in the logs. This must be unique for all steps. **This field is optional** and will be auto-generated if not specified.
-* `tag` this says to only execute this step for the given "tag". Within Codeship this will be the branch or tag name. You can specify this locally via the `--tag` flag. This field is optional.
+* `tag` this field will cause the step to be executed only when the build tag matches the `tag` value. During a Codeship build, the build tag is either the branch name or tag associated with the commit. If you are using this feature locally, you can pass in `jet steps -
+   -tag TAG_NAME` to create a build tag. This field is optional.
```
Comment (mlocher) [inline _posts/docker/2015-05-25-steps.md]: Is the line break on purpose?

Comment (rheinwein) [inline _posts/docker/2015-05-25-steps.md]: Nope :) fixed

**Comment by rheinwein** [comment]
@mlocher I updated the docs to note that `exclude` takes precedence, and added a test case for that in the Jet PR. I intentionally set it up like this because I think it's safer to skip a step than to run it when it wasn't intended to be run. Alternatively we could error the build if both are present, but I'm not sure that's necessary.

**Comment by bfosberry** [comment]
@rheinwein could you take a look at this tomorrow, is this merge-ready?

**Comment by rheinwein** [comment]
@bfosberry see updated description

**Pull request merged**